### ISSUE TITLE
No eyedropper displayed

### ISSUE THREAD

**Issue by knagurski** [superhighfives/pika#71]
_First of all, I just want to say thank you so much for this. Pika is an amazing tool that does EXACTLY what I need!_

I seem to be unable to select a colour. I'm not sure when this started, but the eye dropper doesn't appear, clicking the swatch doesn't trigger it, neither does the keyboard shortcut nor selecting _pick foreground_ or _pick background_ in the menu.

**To Reproduce**
Unsure how to reproduce. It appears this way from launch

**Expected behavior**
The eye dropper icon should appear in each swatch. Clicking this, pressing the keyboard shortcut or selecting _pick foreground_ or _pick background_ should trigger the loop to select a colour.

**Screenshots**

![No eyedropper](https://user-images.githubusercontent.com/7167253/146177881-e5307044-00af-4805-b1d3-75a27d85121f.mp4)

**Environment (please complete the following information):**
 - OS: Mac OS 12.0.1 (21A559)
 - App Version: 0.0.12 (Build 23)
 - Architecture: Intel

**Additional context**
I have tried uninstalling and reinstalling and wiping my app preferences. I've also tried granting screen recording privileges in the system preferences.

**Comment by superhighfives** [comment]
First off, thanks for the kind words @knagurski! That's definitely a doozy of a bug. I haven't tested Monterey with Pika yet, so this is a great excuse to set it up and try to recreate the issue you've come across. Thanks for flagging. 👍

**Comment by superhighfives** [comment]
Hmmm, so, I'm on Monterey (v12.1) and I'm not able to recreate this issue. 🤔 So, possibilities are:
- it's specific to macOS 12.0.1
- there's something else clashing with Pika (potentially another application that you have installed, like [this example](https://twitter.com/cameronmcnab/status/1468316350233726976))
- something else entirely

Will set up 12.0.1 and see if I can recreate—in the meantime, if anyone else has come across this issue please let me know.

**Comment by superhighfives** [comment]
Hmmm, tried it in 12.0.1 too, @knagurski, and I can't seem to recreate it. Is there anything you might have installed or changed that could be conflicting with Pika? Thinking a screenshotting app, or similar.

**Comment by ryelle** [comment]
I'm running into this same problem - picking a color doesn't trigger the eyedropper. I've been having this issue for a while but didn't realize it was consistent, so unfortunately I'm not sure what changed to cause it. I do have ScreenFlow installed, and it's possible Pika stopped working around that time, but I don't see any overlapping shortcuts. Are there any known conflicts between those two?

My environment:

- OS: Mac OS 12.4
- App Version: 0.0.12 (also went back and tried 8 through 13-beta4)
- Architecture: Intel

**Comment by superhighfives** [comment]
Thanks for sharing, @ryelle—it's possible that something is blocking the API that Pika uses to for colour picking (https://developer.apple.com/documentation/appkit/nscolorsampler) somehow, but I'm not 100% sure. Do you see anything relating to Pika in your Console (it's in `/Applications/Utilities/Console.app`) when you try and activate the colour picker Pika? Have you tried using the global shortcut instead, or the setting cog menu in the app?

**Comment by ryelle** [comment]
Well, I updated to macOS 12.5 and Pika seems to be working again. For what it's worth, I was trying all of the methods — clicking the color, keyboard shortcut, and cog menu, and none had worked. But if I set Pika to disappear while selecting a color, it would disappear… it just didn't trigger the color picker.

I looked in the console but TBH I don't know what I'm looking for -- no errors jumped out at me though.

Hopefully Pika will keep working for me now, but I'll try to pay attention to a cause if it stops again. Thanks for all your work on the app!
